### PR TITLE
feat(QUAL-035): 212ファイル復旧差分分析システム実装・239ファイル処理方針確定

### PR DESCRIPTION
**Pull request by miyashita337**
## 📊 Summary
QUAL-034復旧作業で残った239ファイル（stash@{0}）の処理方針を自動分析・決定するシステムを実装。

### 🎯 主要成果
- **自動分析完了**: 239ファイルの変更内容を完全分析
- **処理方針明確化**: 削除推奨201ファイル、統合推奨38ファイル
- **高信頼度判定**: 平均信頼度0.87で安全な処理方針決定
- **テスト完全通過**: 33/33テスト全通過（ユニット27+統合6）

### 🔧 Technical Implementation
- **StashDiffAnalyzer**: git stash差分の自動分析エンジン
- **変更分類システム**: 5種類の変更パターン分類（FORMAT_ONLY/FUNCTIONAL/TRACKER_ID/DOCUMENTATION/MIXED）
- **推奨アクション決定**: 信頼度ベースの4段階判定（INTEGRATE/DISCARD/REVIEW/PRESERVE）
- **包括的レポート生成**: JSON/Markdown形式での詳細分析結果出力

### 📈 Analysis Results
```
総ファイル数: 239ファイル
変更行数: +46,476 / -46,584 (net: -108)
平均信頼度: 0.87

アクション分布:
├─ 削除推奨: 201ファイル (84.1%) - フォーマットのみ変更
├─ 統合推奨: 38ファイル (15.9%) - ドキュメント更新
├─ 要レビュー: 0ファイル (0.0%)
└─ 保留: 0ファイル (0.0%)

変更タイプ分布:
├─ FORMAT_ONLY: 201ファイル (84.1%)
├─ DOCUMENTATION: 38ファイル (15.9%)
├─ FUNCTIONAL: 0ファイル (0.0%)
├─ TRACKER_ID: 0ファイル (0.0%)
└─ MIXED: 0ファイル (0.0%)
```

### 🧪 Test Coverage
- **Unit Tests**: 27 tests covering core functionality
- **Integration Tests**: 6 tests covering real-world scenarios
- **Edge Case Handling**: Binary files, empty changes, large diffs
- **Performance Validation**: 100-file analysis under 10 seconds

### 📋 Next Steps
1. **CI確認**: GitHub Actions実行監視
2. **抽出パイプライン実行**: QUAL-035ワークスペース出力
3. **統計分析**: QUAL-032ベースライン比較
4. **ダッシュボード生成**: 分析結果可視化

🤖 Generated with [Claude Code](https://claude.ai/code)

### CODE REVIEW

**Comment by miyashita337** [comment]
tests/integration/test_stash_analysis_integration.py
tests/unit/test_stash_diff_analyzer.py

このテストは

今後
```

● 素晴らしい分析結果です！239ファイルが分析され：
  - 削除推奨: 201ファイル（フォーマットのみの変更）
  - 統合推奨: 38ファイル（ドキュメント更新）
  - 要レビュー: 0ファイル
```

201ファイルを削除する予定ですが、それにより影響が出ますか？
テストにエラーが生じます？

**Comment by miyashita337** [comment]
## ✅ 推奨アクション実行完了

ご指示いただいた通り、全ての推奨アクションを実行完了しました。

### 📁 実施内容

#### 1️⃣ QUAL-032関連ファイル移動完了
```bash
tools/trackers/QUAL_032_tracker_standardization/
├── __init__.py
├── execute_live_replacement.py
├── lightweight_replacement_result.json
├── lightweight_tracker_replacement.py
├── replacement_dry_run_results.json
├── staged_replacement.py
├── tracker_id_functional_analysis.py
├── tracker_id_mapping.json (参照用コピー)
├── tracker_id_mapping_generator.py
├── tracker_replacement_engine.py
└── verification_checker.py
```

#### 2️⃣ QUAL-035関連ファイル移動完了
```bash
tools/trackers/QUAL_035_stash_analysis/
├── __init__.py
├── stash_diff_analyzer.py
├── stash_analysis_detailed.json
├── stash_analysis_summary.json
└── stash_analysis_report.md
```

#### 3️⃣ stash@{0}削除完了
- **削除実行**: `git stash drop stash@{0}`
- **削除ID**: 06f5634b15948318b248c440e2256df82199c1cd
- **削除内容**: 201ファイル（FORMAT_ONLYタイプ）

### ✅ 動作確認

#### テスト実行結果
```
============================== 33 passed in 0.64s ==============================
```
- 全33テスト（ユニット27 + 統合6）正常通過
- インポートパス変更後も問題なし

#### 変更差分
- **21 files changed**: 799 insertions(+), 353 deletions(-)
- **主な変更**: ファイル移動とインポートパス更新

### 📋 追加実施事項
- Pythonパッケージ化: 各ディレクトリに`__init__.py`追加
- ディレクトリ名修正: ハイフン → アンダースコア（Python準拠）
- 参照データ保持: `tracker_id_mapping.json`を両ディレクトリで利用可能に

### 🎯 今後の推奨アクション
1. **即時**: PR #42のマージ（CIパス確認後）
2. **1ヶ月後**: QUAL-032関連ファイルの最終削除検討
3. **永続保持**: QUAL-035分析ツールと参照データ

実行完了しました。ご確認お願いいたします。

🤖 Executed with Claude Code

File: .gitignore [hunk 1]
```diff
@@ -100,4 +100,23 @@ test_batches/
...
+# Backup files
+*.backup
+*.nav_backup
+
+# Documentation reports
+tracker_id_bulk_replacement_report.md
+tracker_standardization_backup.md
+
+# Deprecated directory
+deprecated/
```
Comment (miyashita337) [inline .gitignore:122]: deprecated
はgitignoreから外してください
もし誤って削除されたとしても、commitの中に入ってる場合は復旧が楽なため

**Comment by miyashita337** [comment]
## ✅ 追加ファイル移動完了

ご指摘ありがとうございます。ルートディレクトリに残っていた3ファイルも移動完了しました。

### 📁 追加移動ファイル

```bash
# 移動前（ルートディレクトリ）
./bulk_tracker_replacement.py
./complete_google_sheets_update.py  
./tracker_function_mapping.py

# 移動後（QUAL-032ディレクトリ）
tools/trackers/QUAL_032_tracker_standardization/
├── bulk_tracker_replacement.py      ✅ NEW
├── complete_google_sheets_update.py ✅ NEW
├── tracker_function_mapping.py      ✅ NEW
└── ... （既存の移動済みファイル）
```

### 📊 最終状態

#### QUAL-032関連ファイル（全14ファイル）
```
tools/trackers/QUAL_032_tracker_standardization/
├── __init__.py
├── bulk_tracker_replacement.py
├── complete_google_sheets_update.py
├── execute_live_replacement.py
├── lightweight_replacement_result.json
├── lightweight_tracker_replacement.py
├── replacement_dry_run_results.json
├── staged_replacement.py
├── tracker_function_mapping.py
├── tracker_id_functional_analysis.py
├── tracker_id_mapping.json
├── tracker_id_mapping_generator.py
├── tracker_replacement_engine.py
└── verification_checker.py
```

#### QUAL-035関連ファイル（変更なし）
```
tools/trackers/QUAL_035_stash_analysis/
├── __init__.py
├── stash_diff_analyzer.py
├── stash_analysis_detailed.json
├── stash_analysis_summary.json
└── stash_analysis_report.md
```

### ✅ 確認事項
- ルートディレクトリにQUAL-032関連ファイルは残っていません
- 全てのファイルが適切なトラッカー別ディレクトリに整理されました
- コミット: 45f7f2c

ご確認をお願いいたします。

🤖 Fixed with Claude Code
